### PR TITLE
Adding aubo_robot to documentation index for distro

### PR DESCRIPTION
**Pull request by liuxinwust**
I'd like aubo_robot to be indexed and documented on ros.org

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -466,6 +466,31 @@ repositories:
...
+      - aubo_description
+      - aubo_driver
+      - aubo_gazebo
+      - aubo_i5_moveit_config
+      - aubo_kinematics
+      - aubo_msgs
+      - aubo_robot
+      - aubo_trajectory
+      tags:
+        release: release/indigo/{package}/{version}
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: We highly recommend to use a separate release repo instead of doing the release in the same repo.

**Comment by auboliuxin** [comment]
add aubo_robot to Indigo distribution.yaml

File: indigo/distribution.yaml (hunk 1 above)
Comment (auboliuxin) [inline indigo/distribution.yaml]: Hello,we  bloom to use a separate release repo(aubo_robot-release or rosdistro),but not success.
cmd line:bloom-release --rosdistro indigo --track indigo aubo_robot --edit.
thanks.

**Comment by tfoote** [comment]
You seem to have run bloom against your ros/rosdistro repository or merged content from your release repo into this repository. There are several other pull requests that replace this so I'm going to close this.

**Pull request closed without merging**